app/controllers/slot_ranges_controller.rb

```diff
@@ -18,13 +18,19 @@ def update
       @slot_range.slots.destroy_all
       @slot_range.update(slot_range_parameters)
     end
-    redirect_to edit_user_path(@guider), flash: { success: "Guider \"#{@guider.name}\" has been updated" }
+    redirect_to(
+      edit_user_path(@guider),
+      flash: { success: "Slot Range \"#{@slot_range.display_title}\" has been updated" }
```

Comment:
Put this in the `ApplicationController` - `add_flash_types :success` and then you can:

```ruby
redirect_to edit_user_path(@guider), success: "Slot Range \"#{@slot_range.display_title}\" has been updated" }
```